src/main/java/Paimon.java

```diff
@@ -0,0 +1,180 @@
+package main.java;
+
+import main.java.task.*;
+
+import java.util.Scanner;
+
+public class Paimon {
+    /**
+     * Default function for sending messages, includes text formatting
+     */
+    private static void sendTaskMessage(String mainMessage, String subMessage, String closingMessage) {
+        System.out.println(mainMessage);
+        System.out.println("-------------------->");
+        System.out.println(subMessage);
+        System.out.println("-------------------->");
+        System.out.println(closingMessage);
+    }
+
+    private static void sendTaskMessage(String mainMessage, String subMessage) {
+        System.out.println(mainMessage);
+        System.out.println("-------------------->");
+        System.out.println(subMessage);
+        System.out.println("-------------------->");
+    }
+
+    private static void printTaskList(TaskList list) {
+        if (list.getSize() == 0) {
+            System.out.println("Your list is currently empty, add some tasks!");
+        } else {
+            System.out.println("Here is your list so far!");
+            System.out.println("-------------------->");
+            System.out.println(list);
+            System.out.println("-------------------->");
+        }
+
+    }
+
+    private static void greeting() {
```

Comment:
Method names should be verbs, so an example would be changing this to `printGreeting`. Similar for `goodbye`.